src/common/api/operations.ts

```diff
@@ -2280,3 +2280,25 @@ export const createAccountWithCreditKey = async (
     return err;
   }
 };
+
+export const claimAccount = async (username: string, key: PrivateKey | any): Promise<any> => {
+  const op: any = [
+    "claim_account",
+    {
+      fee: {
+        amount: "0",
+        precision: 3,
+      },
+      creator: username,
+      extensions: []
+    }
+  ]
```

Comment:
> Please, forgot `any` type and create meaningful ones. If you don't know response type better to use `unknown`

Yes, actually because it was initial commit, I will declare types properly.